lib/OrganizationCard/InterfacesCard.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get, isEmpty } from 'lodash';
+import { AppIcon } from '@folio/stripes/core';
+import { Field } from 'redux-form';
+import {
+  Button,
+  Card,
+  Layout,
+  MultiColumnList,
+  Icon,
+} from '@folio/stripes/components';
+import OrganizationPicker from './OrganizationPicker';
+
+export default class InterfacesCard extends React.Component {
```

Comment:
So this isn't really just an InterfacesCard is it? It renders the entire Organization card...and may list no organizations whatsoever. Can we clean up the way all these components relate to one another? It's super confusing.

Reply:
Also, this component is too big. It renders two different kinds of cards that do different things because one of them is in a Field and one isn't.